.env

```diff
@@ -0,0 +1,2 @@
+REACT_APP_APIENDPOINT="http://localhost:4500/api"
+SKIP_PREFLIGHT_CHECK=true
```

Comment:
please revert this file, we are not supposed to commit the env files as it contains sensitive data.